### PR TITLE
Add "intended" as option in account dropdown properties

### PR DESCRIPTION
**Pull request by marcomessa**
Account component currently supports intended redirection, but the option is missing from component's dropdown.

### CODE REVIEW

File: components/Account.php [hunk 1]
```diff
@@ -58,7 +58,7 @@ public function defineProperties()
 
     public function getRedirectOptions()
     {
-        return [''=>'- refresh page -', '0' => '- no redirect -'] + Page::sortBy('baseFileName')->lists('baseFileName', 'baseFileName');
+        return [''=>'- refresh page -', '0' => '- no redirect -', 'intended' => '- intended url -'] + Page::sortBy('baseFileName')->lists('baseFileName', 'baseFileName');
```
Comment (LukeTowers) [inline components/Account.php:61]: please put each option on its own line.

Comment (marcomessa) [reply components/Account.php:61]: Surely! I apologize but this is current file indentation. Is this ok?

```php 
return [
    '' => '- refresh page -',
    '0' => '- no redirect -',
    'intended' => '- intended url -',
] + Page::sortBy('baseFileName')->lists('baseFileName', 'baseFileName');
```

**Comment by bennothommo** [comment]
@marcomessa Unless I'm mistaken, this appears to be half-finished. You still need to actually make the `intended` redirect happen if that redirect option is selected.

**Comment by marcomessa** [comment]
Hi @bennothommo, the redirect is already implemented. What is missing is the menu item. Your can faind in makeRedirection method:

```php
 /**
     * Redirect to the intended page after successful update, sign in or registration.
     * The URL can come from the "redirect" property or the "redirect" postback value.
     * @return mixed
     */
    protected function makeRedirection($intended = false)
    {
        $method = $intended ? 'intended' : 'to';

        $property = $this->property('redirect');

        if (!strlen($property)) {
            return;
        }

        $redirectUrl = $this->pageUrl($property) ?: $property;

        if ($redirectUrl = post('redirect', $redirectUrl)) {
            return Redirect::$method($redirectUrl);
        }
    }
```

**Comment by LukeTowers** [comment]
@marcomessa did you test that? `$property` would become equal to `intended` if that option was set and it doesn't look like that is checked for in that method.

**Comment by marcomessa** [comment]
@LukeTowers  of course I did, here's a little video if you want to check. I'm not Steven Spielberg, I hope it'll be clear :D


https://user-images.githubusercontent.com/4236142/107748646-ada7d700-6d19-11eb-92a2-a76b7956e84a.mp4

**Comment by daftspunk** [comment]
The intended functionality is already implemented by default, so there is no need to explicitly specify it (unless you want to disable it).

Regarding intended redirects, you do need to select a page to use when the intended page is not found in the session, this is what you choose when you select a page in the dropdown list.

Hope this helps. If you need further clarification, feel free to send us an email

Thanks

**Pull request closed without merging**